### ISSUE TITLE
unable to commit On Vigilance in Besa corpus on gitdox

### ISSUE THREAD

**Issue by ctschroeder** [gucorpling/gitdox#134]
I think I am able to commit everything else.  This might be a resurgence of the strange character issue affecting committing?  I can't figure out why I can't commit this document.  I've made a ton of changes though so I would like to commit.  Thank you for your help!

**Comment by ctschroeder** [comment]
FYI also cannot export this file as SGML and cannot validate.  So I can't preserve the data and can't figure out what the problem is with the file.  @amir-zeldes @lgessler Many thanks!

**Comment by amir-zeldes** [comment]
Thanks for reporting. It looks like the `tok` column header was blank, which caused this problem. This is one of those places where the system relying on the generic EtherCalc interface makes things rather brittle and convention dependent... 

I applied a hardwired fix for this in ac9c0b359611dabf1ab49e95c91aaddb9005eb27 but ultimately it may be worth thinking about the exact nature of the data model constraints we need to impose and making sure they are enforced programmatically. We have some other priorities for the summer, but if there's time it's definitely something to keep in mind. I committed the document for now.
